Mathlib/Combinatorics/SimpleGraph/Connectivity.lean

```diff
@@ -2062,6 +2066,25 @@ lemma mem_coe_supp_of_adj {v w : V} {H : Subgraph G} {c : ConnectedComponent H.c
   rw [← (mem_supp_iff _ _).mp h.1]
   exact ⟨connectedComponentMk_eq_of_adj <| Subgraph.Adj.coe <| h.2 ▸ hadj.symm, rfl⟩
 
+lemma univ_eq_union_supp : (Set.univ : Set V) = ⋃ (c : G.ConnectedComponent), c.supp := by
```

Comment:
`⋃` is called `iUnion`. Also your LHS is simpler than your RHS
```suggestion
lemma iUnion_supp : ⋃ c : G.ConnectedComponent, c.supp = (Set.univ : Set V) := by
```
or even
```suggestion
lemma iUnion_supp : ⋃ c : G.ConnectedComponent, c.supp = Set.univ := by
```
if it works